rsocket-core/src/main/java/io/rsocket/core/RSocketServer.java

```diff
@@ -223,6 +225,22 @@ public RSocketServer maxInboundPayloadSize(int maxInboundPayloadSize) {
     return this;
   }
 
+  /**
+   * Specifies timeout for the first incoming frame from the accepted connection.
+   *
+   * <p>By default this is set to 1 minute.
+   *
+   * @param timeout duration
+   * @return the same instance for method chaining
+   */
+  public RSocketServer setupHandlingTimeout(Duration timeout) {
```

Comment:
How about calling this `maxTimeToFirstFrame`?

Reply:
Sound good to me!

Reply:
```suggestion
  public RSocketServer maxTimeToFirstFrame(Duration timeout) {
```